File: snippets/java/clear_payload.java
import java.util.List;

import static io.qdrant.client.PointIdFactory.id;

import io.qdrant.client.QdrantClient;
import io.qdrant.client.QdrantGrpcClient;

QdrantClient client = new QdrantClient(
                QdrantGrpcClient.newBuilder("localhost", 6334, false).build());

client.clearPayloadAsync("{collection_name}", List.of(id(0), id(3), id(100)), null, null, null)
                .get();

File: snippets/go/clear_payload.go
package client

import (
	"context"

	"github.com/qdrant/go-client/qdrant"
)

func clearPayload() {
	client, err := qdrant.NewClient(&qdrant.Config{
		Host: "localhost",
		Port: 6334,
	})
	if err != nil {
		panic(err)
	}

	_, err = client.ClearPayload(context.Background(), &qdrant.ClearPayloadPoints{
		CollectionName: "{collection_name}",
		Points:         qdrant.NewPointsSelector(qdrant.NewIDNum(0), qdrant.NewID("3"), qdrant.NewIDNum(100)),
	})
	if err != nil {
		panic(err)
	}
}
